### PR TITLE
AKS guidance revisions

### PR DESCRIPTION
**Pull request by ibersanoMS**
# Overview/Summary
Revised AKS guidance by:
- Consolidating duplicates or similar guidance
- Removing broad recommendations that are not specific to AKS
- Added category column to table of contents

## Related Issues/Work Items
None

## This PR fixes/adds/changes/removes
Overall:
- Added category column to table of contents

Consolidated:
- 1 & 10 into AKS-1 with previous AKS-10's query
- 2 & 17 into AKS-2 with previous AKS-17's query 
- 3 & 12 into AKS-3 with AKS-3's query
- 9 & 22 into AKS-15 
- 18 & 20 into AKS-13

Removed
- 6 (Too broad and included in WAF guidance)
- 13 (Too broad)

Updated:
- 9 (now AKS-5) (Updated guidance text)

### Breaking Changes

None

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Comment by ibersanoMS** [comment]
None of the queries have changed, but the numbers in the guidance have shifted because I consolidated and removed guidance items. Let me know if you'd like screenshots of the ones that say they've changed because I updated the guidance number.

File: docs/content/services/container/aks/_index.md [hunk 1]
```diff
@@ -226,13 +199,13 @@ AKS is increasingly being used for stateful applications that require a backup s
...
 
-{{< code lang="sql" file="code/aks-7/aks-7.kql" >}} {{< /code >}}
+{{< code lang="sql" file="code/aks-6/aks-6.kql" >}} {{< /code >}}
 
 {{< /collapse >}}
 
 <br><br>
 
-### AKS-8 - Plan an AKS version upgrade
+### AKS-7 - Plan an AKS version upgrade
```
Comment (ejhenry) [inline docs/content/services/container/aks/_index.md:208]: Could we change this recommendation to evaluate whether the AKS version is supported? Or, if that is already the purpose, we could be more direct in the wording of the recommendation to make clear the cluster is not on a supported version.

File: docs/content/services/container/aks/code/aks-2/aks-2.kql
[the remark this answers is not in the document]
Comment (ibersanoMS) [reply docs/content/services/container/aks/code/aks-2/aks-2.kql:1]: The query returns any nodepools that do not have any taints. A system nodepool that is isolated would have the "CriticalAddonsOnly=true:NoExecute" taint and user nodepools could have any type of taint that the user defines that identities that workload should be running on that nodepool.

**Comment by rodrigosantosms** [comment]
Approved from my side.

**Review by rodrigosantosms — APPROVED** [review]
Approved

File: docs/content/services/container/aks/_index.md (hunk 1 above)
Comment (ibersanoMS) [reply docs/content/services/container/aks/_index.md:208]: Your point is valid and I can reword the guidance to make it clear the cluster needs to be on a supported version, but there's no way to dynamically grab the AKS versions that are currently supported within the ARG query. I can include instructions for someone to check their cluster versions against the current supported ones.

**Review by ejhenry — APPROVED** [review]
LGTM 👍 thanks @ibersanoMS!

**Pull request merged**